src/popup/utils/utils.ts

```diff
@@ -136,17 +138,24 @@ export function watchUntilTruthy<T>(getter: () => T): Promise<NonNullable<T>> {
 export function rxJsObservableToVueState(
   observable: Observable<any>,
   defaultState: any = null,
-): Ref<any> {
+): Ref<any> | any[] {
```

Comment:
This function returns reference. From what I see there's no way that this function could return `any[]`.